src/App.jsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import './App.css'
+import play from './access/paly.svg'
+import pause from './access/pause.svg'
+import resetTime from './access/time.svg'
+import arrow from './access/arrow.svg'
+import {connect} from "react-redux";
+import {addBreakLength, addSessionLength, reset, startStopTimer} from "./redux/appReduxer";
+import {getFormate} from "./utils";
+
+let Setting = ({title, value, upWork, upRest, downWork, downRest, category}) => {
+  let onUpArrow = () => {
+    category === 'work' ? upWork() : upRest();
+  }
+  
+  let onDownArrow = () => {
+    category === 'work' ? downWork() : downRest();
+  }
+  return (
+    <div className='setting'>
+      <div className='settingTitle'>{title}</div>
+      <div className='withArrows'>
+        <img src={arrow} alt="upValue" onClick={onUpArrow} className="upValue"/>
+        <div className='settingValue'>{value}</div>
+        <img src={arrow} alt="downValue" onClick={onDownArrow} className="downValue"/>
+      </div>
+    </div>
+  )
+}
+
+let ButtonWithSvg = ({svg, alt, onClick}) => {
+  return <img className='buttonImg' onClick={onClick || null} src={svg} alt={alt}/>
+}
+
+class App extends React.Component {
+  render() {
+    let {breakLength, sessionLength, currentTimeSession, currentTimeBreak, isProcess, currentMode} = this.props;
+    let isRest = currentMode === 'rest';
+    
+    let currentTimeSessionMin = getFormate(currentTimeSession)
```

Comment:
If it bloody confusing when you look at code and see there are a lot of ``let`` expressions and it makes sense that all of them could be changed.

Make a rule. Use let only for the things that's gonna be changed over time, and use const for things that's not gonna be changed over time at all

Reply:
Accepted, understood